lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -999,6 +999,14 @@
     <datatype extension="gds" type="galaxy.datatypes.tabular:CSV" subclass="true" description="CSV representing GDS parameters."/>
     <datatype extension="feff" type="galaxy.datatypes.tabular:CSV" subclass="true" description="CSV representing summary of FEFF paths."/>
     <datatype extension="feffit" type="galaxy.datatypes.data:Text" subclass="true" description="Report of Artemis FEFFIT."/>
+    <!-- GSASII types-->
+    <datatype extension="gpx" type="galaxy.datatypes.gsas2:Gsas2Project" display_in_upload="true" description="GSASII project file." />
+    <datatype extension="prm" type="galaxy.datatypes.gsas2:InstrumentParameter" display_in_upload="true" description="GSASII prm instrument parameter file." />
+    <datatype extension="instprm" type="galaxy.datatypes.gsas2:InstrumentParameter2" display_in_upload="true" description="GSASII instprm instrument parameter file." />
+    <datatype extension="raw" type="galaxy.datatypes.gsas2:RawPowderData" display_in_upload="true" description="GSASII raw neutron powder diffraction data" />
```

Comment:
Does this only apply to neutrons? Doesn't GSAS support powder diffraction of X-rays too?